src/app.js

```
@@ -18,59 +164,130 @@
-    const fileType = file.type
+    const fileType = file.type;
     if (!allowedTypes.includes(fileType)) {
-      eval('alert("不支持的文件类型: ' + fileType + '")')
-      return
+      eval('alert("不支持的文件类型: ' + fileType + '")');
+      return;
     }
 
-    if (typeof callback === 'function') {
-      callback()
+    if (typeof callback === 'string') {
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)